src/components/Releases/ReleasePipelineRunTab.tsx

```diff
@@ -1,133 +1,158 @@
 
-  if (!pipelineRuns || pipelineRuns.length === 0) {
-    return <PipelineRunEmptyState applicationName={applicationName} />;
-  }
+  const allRuns: PipelineRunProcessing[] = [
+    createPipelineRunEntry(
+      'Tenant Collector',
+      getTenantCollectorProcessingFromRelease(release),
+      release.spec.snapshot,
+    ),
+    createPipelineRunEntry(
+      'Tenant',
+      getTenantProcessingFromRelease(release),
+      release.spec.snapshot,
+    ),
+    createPipelineRunEntry('Final', getFinalFromRelease(release), release.spec.snapshot),
+    createPipelineRunEntry(
+      'Managed',
+      getManagedProcessingFromRelease(release),
+      release.spec.snapshot,
+    ),
+  ].filter((run) => run !== undefined && run !== null);
+
+  const { name: nameFilter } = filters;
+
+  const filteredRuns = allRuns.filter((run) =>
+    run.pipelineRun.toLowerCase().includes(nameFilter.toLowerCase()),
+  );
 
-  const DataToolbar = (
-    <BaseTextFilterToolbar
-      text={nameFilter}
-      label="name"
-      setText={(name) => setFilters({ name })}
-      onClearFilters={onClearFilters}
-      dataTest="release-pipeline-run-list-toolbar"
-    />
+  const NoDataEmptyMessage = () => (
+    <AppEmptyState
+      emptyStateImg={emptyStateImgUrl}
+      title="No linked secrets found"
+      data-test="linked-secrets-list-no-data-empty-message"
+    >
+      <EmptyStateBody>This release has no pipeline runs yet.</EmptyStateBody>
+    </AppEmptyState>
   );
 
+  const EmptyMessage = () => <FilteredEmptyState onClearFilters={onClearFilters} />;
+
   return (
     <>
       <Title headingLevel="h3" className="pf-v5-c-title pf-v5-u-mt-lg pf-v5-u-pl-lg">
         Pipeline runs
       </Title>
       <Table
-        data-test="release-pipeline-run__table"
-        data={filteredPipelineRuns}
-        aria-label="Pipeline Run List"
-        Header={PipelineRunListHeaderForRelease}
-        Row={PipelineRunListRowForRelease}
-        Toolbar={DataToolbar}
-        loaded
-        customData={{
-          releaseName,
-          releasePlan,
-          release,
-        }}
-        getRowProps={(obj: PipelineRunKind) => ({
-          id: obj.metadata.uid,
+        data={filteredRuns}
+        unfilteredData={allRuns}
+        EmptyMsg={EmptyMessage}
+        NoDataEmptyMsg={NoDataEmptyMessage}
+        Toolbar={
+          <BaseTextFilterToolbar
+            text={filters.name}
+            label="name"
+            setText={(name) => setFilters({ ...filters, name })}
+            onClearFilters={onClearFilters}
+            data-test="release-pipeline-runs-toolbar"
+          />
+        }
+        aria-label="release-pipeline-runs-table"
+        Header={ReleasePipelineListHeader}
+        Row={(props) => (
+          <ReleasePipelineListRow
+            {...props}
+            obj={props.obj as PipelineRunProcessing}
+            releasePlan={releasePlan}
+            releaseName={releaseName}
+            namespace={namespace}
+            status={status}
```

Comment:
Remove status column, because if one of the pipeline run fails and all other passes, release status will be failed and you will report fail for all pipeline runs.